docker/README.rst

```diff
@@ -0,0 +1,173 @@
+Docker Setup for FermiLib + ProjectQ
+====================================
+
+This Docker image will help users to easily install `FermiLib <https://github.com/ProjectQ-Framework/FermiLib.git>`__ and `ProjectQ <https://github.com/ProjectQ-Framework/ProjectQ>`__. Check out Docker's `website <https://www.docker.com/what-container>`__ that describes what a container image is and why it can be so useful.
+
+What is included?
+-----------------
+
+- Python 2.7 (you can also use Python 3 with one minor change in the Dockerfile. See the Dockerfile for instructions.)
```

Comment:
Dockerfile -> dockerfile. Also there should be a link to it. Again, I think its best to add to our old instructions instead of replacing them entirely.

Reply:
I have added back the old instructions, with my stuffs starting from "Setting up Docker for the first time". From the latest README.md, one issue I see is that the section "Running OpenFermion in Docker" I wrote has a large overlap with the old "Usage" section. My section is a bit more hand-holding and covers slightly more common issues that I came across when I was using Docker (and I imagine other people will run into these issues too e.g. how to open more than one terminals running the same container). But feel free to edit the README however you see fit. I don't have particular insistence on any of the content being there.